### PR TITLE
replace agrc to ugrc where applicable with proper yml formatting

### PR DESCRIPTION
**Pull request by stdavis**
This re-applies the reverted name change commit. We can work on this branch to get all of the formatting fixed.

### CODE REVIEW

File: _posts/2019-06-11-sgid-then-and-now.md [hunk 1]
```diff
@@ -28,31 +28,31 @@ In 1980, ESRI was hired as a consultant to assist with Utah’s GIS implementati
...
+The SGID was further solidified in 1991 with the passage of [Senate Bill 21](https://le.utah.gov/xcode/Title63F/Chapter1/63F-1-S507.html).  Passing unanimously, this bill created the UGRC and the SGID in state statute and eliminated many uncertainties such as future funding for the database and UGRC’s responsibilities.
 
 The SGID has been in continuous operation since its completion in 1991 and has seen numerous software and hardware configurations throughout the years.
 
 Here are a few of the highlights that have brought the SGID to where it is now:
 
 - **SGID User's Guide - 7th Edition (1994)**
-  - This user guide was distributed in a three-ring binder. Data was then requested in the following available formats: ARC/INFO Coverages, USGS DLG, MOSS Export, and AutoCAD DXF. This guide also makes note of an "AGRC home page on the world wide web," which was set to debut in November 1994.
+  - This user guide was distributed in a three-ring binder. Data was then requested in the following available formats: ARC/INFO Coverages, USGS DLG, MOSS Export, and AutoCAD DXF. This guide also makes note of an "UGRC home page on the world wide web," which was set to debut in November 1994.
```
Comment (jacobdadams) [inline _posts/2019-06-11-sgid-then-and-now.md:40]:
```suggestion
  - This user guide was distributed in a three-ring binder. Data was then requested in the following available formats: ARC/INFO Coverages, USGS DLG, MOSS Export, and AutoCAD DXF. This guide also makes note of an "AGRC home page on the world wide web," which was set to debut in November 1994.
```
As a quote, this probably best should be left as AGRC.

File: _posts/2021-02-19-intergenerational-poverty.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ Using spatial tools like suitability analysis, we can highlight areas on a map t
...
 
-> Take a look AGRC’s [**interactive neighborhood suitability map**]({% link developer/applications/suitability/index.html %}) to further explore the map below. The layers in this map give the user the ability to discover areas in the state where residents do and do-not have access to essential services, such as childcare, hospitals, grocery stores, libraries, and transit stops. This map shows a sample of what can be done using GIS and spatial analysis.
+> Take a look UGRC’s [**interactive neighborhood suitability map**]({% link developer/applications/suitability/index.html %}) to further explore the map below. The layers in this map give the user the ability to discover areas in the state where residents do and do-not have access to essential services, such as childcare, hospitals, grocery stores, libraries, and transit stops. This map shows a sample of what can be done using GIS and spatial analysis.
```
Comment (jacobdadams) [inline _posts/2021-02-19-intergenerational-poverty.md:37]:
```suggestion
> Take a look at UGRC’s [**interactive neighborhood suitability map**]({% link developer/applications/suitability/index.html %}) to further explore the map below. The layers in this map give the user the ability to discover areas in the state where residents do and do-not have access to essential services, such as childcare, hospitals, grocery stores, libraries, and transit stops. This map shows a sample of what can be done using GIS and spatial analysis.
```

File: about/policy/sgid/index.md [hunk 3]
```diff
@@ -113,4 +113,4 @@ We anticipate some SGID datasets won't receive future updates, such as the bound
 ### "Shelved" Datasets
 {: .text-left }
 
-Over time, we will remove datasets from the SGID geodatabase that are no longer current but continue to have historical value (e.g., Census/ACS data or previous tax district boundaries). Their associated ArcGIS Online items will be tagged as `shelved` and placed in the public [AGRC Shelf](http://utah.maps.arcgis.com/home/group.html?id=8765687b7b0545668fff02d1b750f7a9) ArcGIS Online group for future reference.
+Over time, we will remove datasets from the SGID geodatabase that are no longer current but continue to have historical value (e.g., Census/ACS data or previous tax district boundaries). Their associated ArcGIS Online items will be tagged as `shelved` and placed in the public [UGRC Shelf](http://utah.maps.arcgis.com/home/group.html?id=8765687b7b0545668fff02d1b750f7a9) ArcGIS Online group for future reference.
```
Comment (jacobdadams) [inline about/policy/sgid/index.md:116]:
```suggestion
Over time, we will remove datasets from the SGID geodatabase that are no longer current but continue to have historical value (e.g., Census/ACS data or previous tax district boundaries). Their associated ArcGIS Online items will be tagged as `shelved` and placed in the public [AGRC Shelf](http://utah.maps.arcgis.com/home/group.html?id=8765687b7b0545668fff02d1b750f7a9) ArcGIS Online group for future reference.
```
Not yet renamed; I'll edit this once I get it renamed (need to coordinate with Auditor and maybe a few other places)

File: agrc-newsletter/index.html [hunk 4]
```diff
@@ -1,78 +1,324 @@
...
+      >
+    </p>
+    <p>
+      <a
+        href="https://mailchi.mp/5dae64b97c9b/latest-updates-from-the-utah-agrc"
+        >UGRC Newsletter December 2020</a
+      >
+    </p>
+    <p>
+      <a
```
Comment (jacobdadams) [inline agrc-newsletter/index.html:43]: Was this reformatting intentional?

File: data/911/index.html [hunk 5]
```diff
@@ -25,29 +25,44 @@
...
+    <p>Follow us on <a href="https://twitter.com/maputah?lang=en">Twitter</a>, where you will get information about
+      updates. </p>
     <p>There are no constraints or warranties with regard to the use of this dataset.</p>
     <p>{% include contact.html subject=page.title contact=site.data.contacts.roads %}</p>
   </div>
   {% include packagedata.html
-    name="NG911"
-    info=page.NG911
+  name="NG911"
+  info=page.NG911
```
Comment (jacobdadams) [inline data/911/index.html:66]: @eneemann, does this page preview look right?

File: agrc-newsletter/index.html (hunk 4 above)
Comment (stdavis) [reply agrc-newsletter/index.html:43]: IIRC this file was using something like six spaces for tabs. So I ran my formatter. I agree that it looks a little funny, but it's syntactically fine and I felt like it was an improvement.

File: data/911/index.html (hunk 5 above)
Comment (eneemann) [reply data/911/index.html:66]: No...there should be  file geodatabase to download :
image

Should look like:
image

File: data/elevation-and-terrain/index.html [hunk 6]
```diff
@@ -23,7 +23,7 @@
       label: 'Download: Statewide 30 meter USGS DEM'
   updates:
     - January 2018
----
+----
```
Comment (jacobdadams) [inline data/elevation-and-terrain/index.html:26]:
```suggestion
----
```
```suggestion
---
```
Borks page header?

File: data/911/index.html (hunk 5 above)
Comment (stdavis) [reply data/911/index.html:66]: I'll fix this page...

File: data/elevation-and-terrain/index.html (hunk 6 above)
Comment (eneemann) [reply data/elevation-and-terrain/index.html:26]: And it looks like the downloads are gone from the USGS 3DEP section on this page (data/elevation-and-terrain/index.html)

File: data/911/index.html (hunk 5 above)
Comment (stdavis) [reply data/911/index.html:66]: Fixed via: df791adea3888159c7f90d4c0843a1e39077f17e

**Review by eneemann — CHANGES_REQUESTED** [review]
I stumbled onto another page that's missing downloads: data/society/schools-libraries/

File: data/society/schools-libraries/index.html [hunk 7]
```diff
@@ -1,7 +1,7 @@
 ---
-author:
+  author:
```
Comment (jacobdadams) [inline data/society/schools-libraries/index.html:2]:
```suggestion
author:
```
Does this fix downloads?

Comment (stdavis) [reply data/society/schools-libraries/index.html]: Yes, I just pushed this same commit.

Comment (jacobdadams) [reply data/society/schools-libraries/index.html]: 💨

**Comment by stdavis** [comment]
Data pages for review:
- [ ] data/911/index.html 
- [ ] data/address-geocoders-locators/index.html 
- [ ] data/address/address-grids/index.html 
- [ ] data/address/census-luca-2020.md 
- [ ] data/address/index.html 
- [ ] data/aerial-photography/doq/index.html 
- [ ] data/aerial-photography/hro/index.html 
- [ ] data/aerial-photography/imagery-problems/index.md 
- [ ] data/aerial-photography/index.html 
- [ ] data/aerial-photography/naip/index.html 
- [ ] data/base-map-and-imagery/index.md 
- [ ] data/bioscience/department-wildlife-resources-habitat-areas/index.html 
- [ ] data/bioscience/index.html 
- [ ] data/boundaries/citycountystate/index.html 
- [ ] data/boundaries/conservation-districts/index.html 
- [ ] data/boundaries/index.html 
- [ ] data/boundaries/school-districts/index.html 
- [ ] data/boundaries/wilderness/index.html 
- [ ] data/boundaries/zip-codes/index.html 
- [ ] data/cadastre/index.html 
- [ ] data/cadastre/land-ownership/index.html 
- [ ] data/cadastre/parcels/index.html 
- [ ] data/cadastre/plss/index.html 
- [ ] data/cadastre/turn-gps/index.md 
- [ ] data/climate/index.html 
- [ ] data/demographic/census/index.html 
- [ ] data/demographic/index.html 
- [ ] data/economy/enterprise-zones/index.html 
- [ ] data/economy/index.html 
- [ ] data/economy/taxingareas/index.html 
- [ ] data/elevation-and-terrain/2-meter-lidar/index.html 
- [ ] data/elevation-and-terrain/2011-lidar/index.html 
- [ ] data/elevation-and-terrain/2013-2014-lidar/index.html 
- [ ] data/elevation-and-terrain/2016-lidar-blbrcvuwv/index.md 
- [ ] data/elevation-and-terrain/2016-lidar-forge/index.md 
- [ ] data/elevation-and-terrain/2016-lidar-gsl/index.md 
- [ ] data/elevation-and-terrain/2016-lidar-monroe-mtn/index.md 
- [ ] data/elevation-and-terrain/2017-lidar-millsite-res/index.md 
- [ ] data/elevation-and-terrain/2017-lidar-washington-co/index.md 
- [ ] data/elevation-and-terrain/2017-lidar-wvw/index.md 
- [ ] data/elevation-and-terrain/2018-lidar-box-elder/index.md 
- [ ] data/elevation-and-terrain/2018-lidar-central-utah/index.md 
- [ ] data/elevation-and-terrain/2018-lidar-northern-utah/index.md 
- [ ] data/elevation-and-terrain/2018-lidar-sanpete-valley/index.md 
- [ ] data/elevation-and-terrain/2018-lidar-southern-utah/index.md 
- [ ] data/elevation-and-terrain/2019-lidar-kane-county/index.md 
- [ ] data/elevation-and-terrain/2019-lidar-strawberry-river/index.md 
- [ ] data/elevation-and-terrain/contours/index.html 
- [ ] data/elevation-and-terrain/exploring-lidar/index.html 
- [ ] data/elevation-and-terrain/index.html 
- [ ] data/energy/coal/index.html 
- [ ] data/energy/energy-generation/index.html 
- [ ] data/energy/index.html 
- [ ] data/energy/oil-gas/index.html 
- [ ] data/energy/renewable-energy/index.html 
- [ ] data/energy/uranium/index.html 
- [ ] data/environment/deq-air-related-contaminant-cleanup-sites/index.html 
- [ ] data/environment/deq-land-related-contaminant-cleanup-sites/index.html 
- [ ] data/environment/drinking-water-systems-protection-zones/index.html 
- [ ] data/environment/index.html 
- [ ] data/farming/grazing-improvements/index.html 
- [ ] data/farming/index.html 
- [ ] data/geoscience/aquifers/index.html 
- [ ] data/geoscience/avalanche/index.html 
- [ ] data/geoscience/index.html 
- [ ] data/geoscience/landslides/index.html 
- [ ] data/geoscience/soil/index.html 
- [ ] data/health/health-care-facilities/index.html 
- [ ] data/health/health-districts/index.html 
- [ ] data/health/health-small-statistical-areas/index.html 
- [ ] data/health/index.html 
- [ ] data/history/index.html 
- [ ] data/index.html 
- [ ] data/indices/aerial-photography-indices/index.html 
- [ ] data/indices/auto-correlated-dem-indices/index.html 
- [ ] data/indices/contours-indices/index.html 
- [ ] data/indices/index.html 
- [ ] data/indices/lidar-indices/index.html 
- [ ] data/indices/national-grid-index/index.html 
- [ ] data/indices/usgs-topo-indices/index.html 
- [ ] data/location/building-footprint/index.html 
- [ ] data/location/gnis/index.html 
- [ ] data/location/index.html 
- [ ] data/location/u-s-postal-service/index.html 
- [ ] data/planning/index.html 
- [ ] data/planning/water-related-land/index.html 
- [ ] data/political/2002-2011-house-senate-congressional-districts/index.html 
- [ ] data/political/2012-2021-house-senate-congressional-districts/index.html 
- [ ] data/political/index.html 
- [ ] data/political/judicial-districts/index.html 
- [ ] data/political/voter-precincts/index.html 
- [ ] data/recreation/boat-ramps/index.html 
- [ ] data/recreation/golf-courses/index.html 
- [ ] data/recreation/index.html 
- [ ] data/recreation/local-parks/index.html 
- [ ] data/recreation/ski-areas/index.html 
- [ ] data/recreation/trails/index.html 
- [ ] data/sgid-blog/index.html 
- [ ] data/society/blm-field-offices-forest-service-stations/index.html 
- [ ] data/society/cemeteries/index.html 
- [ ] data/society/index.html 
- [ ] data/society/liquor-stores/index.html 
- [ ] data/society/places-of-worship/index.html 
- [ ] data/society/public-safety/index.html 
- [ ] data/society/schools-libraries/index.html 
- [ ] data/society/state-facilities/index.html 
- [ ] data/society/state-fuel-sites/index.html 
- [ ] data/transportation/air/index.html 
- [ ] data/transportation/index.html 
- [ ] data/transportation/railroads/index.html 
- [ ] data/transportation/roads-system/create-network-analysis-dataset-instructions.html 
- [ ] data/transportation/roads-system/index.html 
- [ ] data/transportation/street-network-analysis/index.html 
- [ ] data/transportation/transit/index.html 
- [ ] data/usgs-scanned-topographic-maps/index.html 
- [ ] data/utilities/broadband-internet/index.html 
- [ ] data/utilities/index.html 
- [ ] data/utilities/retail-culinary-water-suppliers/index.html 
- [ ] data/utilities/telephone-infrastructure-and-rural-telcom-boundaries/index.html 
- [ ] data/water/flood-areas/index.html 
- [ ] data/water/historic-lake-bonneville/index.html 
- [ ] data/water/index.html 
- [ ] data/water/lakes-rivers-dams/index.html 
- [ ] data/water/watersheds/index.html 
- [ ] data/water/wetlands/index.html

**Review by jacobdadams — APPROVED** [review]
HTML. HTML everywhere.

**Review by eneemann — APPROVED** [review]

**Pull request merged**